fuel_test/deploy_env.sh

```diff
@@ -0,0 +1,298 @@
+#!/bin/bash
+
+set -eux
+
+function wait_for_fm {
+	# Wait for fuel master booting and return its IP address
+	local xs_host="$1"
+	local fm_name="$2"
+	local retry_count=$3
+	local retry_interval=$4
+
+	local fm_ip
+	local counter=0
+	while [ $counter -lt $retry_count ]; do
+		fm_ip=$(ssh -qo StrictHostKeyChecking=no root@$xs_host \
+	'
+	fm_networks=$(xe vm-list name-label="'$fm_name'" params=networks --minimal)
+	fm_ip=$(echo $fm_networks | egrep -Eo "1/ip: ([0-9]+\.){3}[0-9]+");
+	echo ${fm_ip: 5}
+	')
+		if [ -n "$fm_ip" ]; then
+			echo $fm_ip
+			return
+		fi
+		let counter=counter+1
+		sleep $retry_interval
+	done
+}
+
+function start_node {
+	# Boot up given node
+	local xs_host="$1"
+	local vm="$2"
+	ssh -qo StrictHostKeyChecking=no root@$xs_host \
+	'
+	set -eux
```

Comment:
If it's a single line command in the subshell / ssh shell, just run it - no need for the set -eux.
ssh -qo StrictHostKeyChecking=no root@$xs_host xe vm-start vm="$vm"